### PR TITLE
Feature/netezza plugin tests

### PR DESCRIPTION
**Pull request by ibarbashov**

### CODE REVIEW

File: netezza-plugin/src/test/java/co/cask/netezza/NetezzaPluginTestBase.java [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    prepareTestData(conn);
+  }
+
+  protected static void createTestTables(Connection conn) throws SQLException {
+    try (Statement stmt = conn.createStatement()) {
+      // create a table that the action will truncate at the end of the run
+      stmt.execute("CREATE TABLE db_action_test (x int, day varchar(10))");
+      // create a table that the action will truncate at the end of the run
+      stmt.execute("CREATE TABLE post_action_test (x int, day varchar(10))");
+
```
Comment (jayfjin) [inline netezza-plugin/src/test/java/co/cask/netezza/NetezzaPluginTestBase.java:122]: extra newline

File: netezza-plugin/src/test/java/co/cask/netezza/NetezzaSourceTestRun.java [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+    Assert.assertEquals(expectedTime, row1.getTime("TIME_COL"));
+    Assert.assertEquals(expectedTs, row1.getTimestamp("TIMESTAMP_COL", ZoneId.ofOffset("UTC", ZoneOffset.UTC)));
+
+    // verify binary columns
+    Assert.assertEquals("user1", Bytes.toString(((ByteBuffer) row1.get("VARBINARY_COL")).array(), 0, 5));
+    Assert.assertEquals("user2", Bytes.toString(((ByteBuffer) row2.get("VARBINARY_COL")).array(), 0, 5));
+    Assert.assertEquals("user1", Bytes.toString(((ByteBuffer) row1.get("ST_GEOMETRY_COL")).array(), 0, 5));
+    Assert.assertEquals("user2", Bytes.toString(((ByteBuffer) row2.get("ST_GEOMETRY_COL")).array(), 0, 5));
+
+
```
Comment (jayfjin) [inline netezza-plugin/src/test/java/co/cask/netezza/NetezzaSourceTestRun.java]: extra newline

**Review by jayfjin — COMMENTED** [review]
If there is duplication between this and https://github.com/data-integrations/database-plugins/pull/13 please rebase before submitting

File: netezza-plugin/src/test/java/co/cask/netezza/NetezzaPluginTestBase.java (hunk 1 above)
Comment (ibarbashov) [reply netezza-plugin/src/test/java/co/cask/netezza/NetezzaPluginTestBase.java:122]: removed

File: netezza-plugin/src/test/java/co/cask/netezza/NetezzaSourceTestRun.java (hunk 2 above)
Comment (ibarbashov) [reply netezza-plugin/src/test/java/co/cask/netezza/NetezzaSourceTestRun.java]: removed

**Comment by jayfjin** [comment]
LGTM

**Pull request merged**